karma.conf.ts

```diff
@@ -27,7 +27,10 @@ module.exports = (config) => {
     frameworks: ['jasmine', 'webpack'],
 
     // List of files / patterns to load in the browser.
-    files: ['test/**/*.test.ts'],
+    files: [
+      // Set TEST_PATTERN env var to test a specific file.
+      process.env.TEST_PATTERN || 'test/**/*.test.ts',
```

Comment:
This makes it easy to run a specific test file from the command line without running the whole suite.